src/mongocxx/collection.hpp

```diff
@@ -47,6 +47,8 @@
 #include <mongocxx/result/update.hpp>
 #include <mongocxx/write_concern.hpp>
 
+#include <mongoc.h>
```

Comment:
Should use `#include <mongocxx/private/libmongoc.hpp>`.

Is this okay to do in a public header file? I don't know. I think we haven't, though.